code/headers/st7735.hpp

```diff
@@ -0,0 +1,166 @@
+#ifndef R2D2_ST7735_HPP
+#define R2D2_ST7735_HPP
+
```

Comment:
although not my PR to review please remove include guards for consistency

Reply:
It is my PR, please remove :)

Reply:
This is fixed in the latest commit, So I'm closing this.